agent/04_agent_configure.sh

```diff
@@ -64,6 +64,9 @@ function generate_cluster_manifests() {
   # To be change in ${OCP_DIR}/cluster-manifests
   MANIFESTS_PATH="${OCP_DIR}/manifests"
 
+  # Fetch current OpenShift version from the release payload
+  VERSION="$(openshift_version ${OCP_DIR})"
```

Comment:
I'm seeing an error in my local test of "make agent":
```
++(./agent/04_agent_configure.sh:68): generate_cluster_manifests(): openshift_version ocp/ostest
./agent/04_agent_configure.sh: line 68: openshift_version: command not found

```
I think this needs to be added to the beginning of the file to include the openshift_version function:
`source $SCRIPTDIR/ocp_install_env.sh`

Reply:
Thanks @rwsu